### ISSUE TITLE
Product Backlog development

### ISSUE THREAD

**Issue by FrankBerrocal** [FrankBerrocal/BigSnowMan#36]
Definition of Product backlog.

**Comment by FrankBerrocal** [comment]
Based on Jira Structure:



![Image](https://user-images.githubusercontent.com/66488543/204094458-4f8cf3a2-274a-4c08-88f5-967c5b2fb39e.png)

**Comment by FrankBerrocal** [comment]
I have redesigned the elements of the project element in GitHub, to resemble this structure.  

Now I have three views.  

One that includes the backlog, the current spring and the work done (under the name of that sprint)



![Image](https://user-images.githubusercontent.com/66488543/204094562-4251a27a-150c-4526-8a15-c503c7836816.png)



And another one, where all the complete Sprint are presented.  



![Image](https://user-images.githubusercontent.com/66488543/204094581-7bb83efa-47a4-4cb1-a280-7f8022e92ed6.png)



I have eliminated the column Test.  Column Authorization will remain on the ALL Sprints view.
